ts/Dashboards/EditMode/EditRenderer.ts

```diff
@@ -572,7 +572,7 @@ function renderInput(
             id: options.id || '',
             name: options.name || '',
             value: (
-                options.value && options.value.replace(/\"/g, '') ||
+                options.value && options.value.toString().replace(/\"/g, '') ||
```

Comment:
Something like `isString(options.value) ? ...replace... : value` might be better, because you will miss zero otherwise.

Reply:
Good suggestion will fix it!

Reply:
Good suggestion, I used `defined()` from Utils and it covers the problem.